docs-content/getting-started/fullstack-frameworks/next-js/2_page-router.md

```diff
@@ -43,6 +43,7 @@ export default function MyApp({ Component, pageProps }: AppProps) {
 					enabled: true,
 					recordHeadersAndBody: true
 				}}
+				debug
```

Comment:
Do we want these in the docs?

Reply:
Yeah. It's a nice default so that they initially see the logs and can confirm that the session is connected.